components/atoms/CartIllustration/cart-illustration.tsx

```diff
@@ -3,13 +3,9 @@ interface CartIllustrationProps {
 }
 const CartIllustration = ({ classNames }: CartIllustrationProps) => {
   return (
-    <div
-      className={`inline-flex max-w-[176px] gap-1 items-center px-1.5 w-44 h-12 rounded-md border ${
-        classNames ?? classNames
-      }`}
-    >
-      <div className="h-[34px] w-[34px] border rounded bg-light-slate-5"></div>
-      <div className="w-[80px]  gap-2  rounded">
+    <div className={`inline-flex gap-1 items-center px-1.5 w-44 h-12 rounded-md border ${classNames ?? classNames}`}>
```

Comment:
In the previous version, we have `max-w...` in the current version we don't have it. this is an intended feature ???

Reply:
I removed max-w-[176px] since it duplicates the width set by w-44, causing redundancy in the styling. What do you suggest?